frontend/src/static/js/components/webstatus-overview-content.ts

```diff
@@ -53,6 +65,31 @@ export class WebstatusOverviewContent extends LitElement {
     ];
   }
 
+  renderMappingPercentage(): TemplateResult {
+    if (
+      this.webFeaturesProgress === undefined ||
+      this.webFeaturesProgress.isDisabled
+    ) {
+      return html``;
+    }
+    if (this.webFeaturesProgress.error) {
+      new Toast().toast(
+        this.webFeaturesProgress.error,
+        'danger',
+        'exclamation-triangle'
+      );
+      return html``;
+    }
+    return html`Percentage of Features Mapped:&nbsp;
```

Comment:
Why the non-breaking space?

Reply:
It's a weird scenario. The linter put the rest on the next line. So it was showing the rate as `Percentage of Features Mapped:XX%` (no space after the colon)

Reply:
I am definitely open to any other suggestions to get rid of the nbsp

Reply:
Hmm, I would think that a line break counts as whitespace.  The first alternative that comes to mind is to put the text in a `<span>` and add some padding or margin to the span.  But, using `&nbsp;` is also fine.